src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -89,26 +90,21 @@
         buildPemFileDialog().choose().ifPresent(file -> setPemFile(Paths.get(file.toURI())))
     );
     pemFileClearButton.addActionListener(__ -> clearPemFile());
+
     startDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
-        startDatePicker.clear();
-      } else
-        onSelectStartDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
-    });
-    endDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
+
+      if (event.getNewDate() == null || isInvalidStartDate(extractDate(event)))
```

Comment:
Every call to `extractDate(event)` could be replaced by `event.getNewDate()` because now the datepicker returns a `java.time.LocalDate` directly (we were using an old version before that used a third party library for dates).

As a result, the `extractDate` method could be removed.

Reply:
Done